tests/test_features/test_correction/test_generator.py

```
@@ -244,6 +244,124 @@ def test_validate_counts_only_numbered_lines_with_subheaders(monkeypatch: pytest
+    monkeypatch.setattr(generator, "_load_generator_fallback_max_retries", lambda: 4)
+
+    instance = generator.CorrectionGenerator()
+
+    assert instance._max_retries == 4
+
+
+def test_validate_respects_min_lines_per_field(monkeypatch: pytest.MonkeyPatch):
+    """min_lines_per_field 설정을 검증 로직에 반영한다."""
+    from features.correction import generator
+
+    monkeypatch.setattr(generator, "get_llm", lambda model=None, temperature=0.2: DummyLLM())
```

Ruff (0.15.2)

[warning] Unused lambda argument: `model`

(ARG005)